console/console-init/ui/src/components/MetadataListTable/TableRow.tsx

```diff
@@ -37,31 +38,20 @@ const styles = StyleSheet.create({
   }
 });
 
+interface IRowData {
```

Comment:
can we rename this as there is already a IRowData interface in the code

Reply:
done